JustSaying.Sample.Restaurant.OrderingApi/Startup.cs

```diff
@@ -0,0 +1,109 @@
+                    {
+                        // The AWS client SDK allows specifying a custom HTTP endpoint.
+                        // For testing purposes it is useful to specify a value that
+                        // points to a docker image such as `p4tin/goaws` or `localstack/localstack`
+                        x.WithServiceUrl(_awsServiceUrl);
+                    }
+                    else
+                    {
+                        // The real AWS environment will require some means of authentication
+                        //x.WithBasicCredentials("###", "###");
+                        //x.WithSessionCredentials("###", "###", "###");
+                    }
+                });
+                config.Messaging(x =>
+                {
+                    // Configures which AWS Region to operate in
+                    x.WithRegion(RegionEndpoint.GetBySystemName(_awsRegion));
+                });
+                config.Subscriptions(x =>
+                {
+                    // Creates the following if they do not already exist
+                    //  - a SQS queue of name `orderreadyevent`
+                    //  - a SQS queue of name `orderreadyevent_error`
+                    //  - a SNS topic of name `orderreadyevent`
+                    //  - a SNS topic subscription on topic 'orderreadyevent' and queue 'orderreadyevent'
+                    x.ForTopic<OrderReadyEvent>();
+                });
+                config.Publications(x =>
+                {
+                    // Creates the following if they do not already exist
+                    //  - a SNS topic of name `orderplacedevent`
+                    x.WithTopic<OrderPlacedEvent>();
+                });
+            });
+
+            // Added a message handler for message type for 'OrderReadyEvent' on topic 'orderreadyevent' and queue 'orderreadyevent'
+            services.AddJustSayingHandler<OrderReadyEvent, OrderReadyEventHandler>();
+
+            // Add a background service that is listening for messages related to the above subscriptions
+            services.AddHostedService<Subscriber>();
+
+            services.AddLogging(c =>
+            {
+                c.AddConsole();
+            });
+
+            services.AddSwaggerGen(c =>
+            {
+                c.SwaggerDoc("v1", new Info { Title = "Restaurant Ordering API", Version = "v1" });
+            });
+        }
+
+        public static void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        {
+            app.UseSwagger();
+            app.UseSwaggerUI(c =>
+            {
+                c.SwaggerEndpoint("/swagger/v1/swagger.json", "Restaurant Ordering API");
+                c.RoutePrefix = string.Empty;
+            });
+
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+            else
+            {
+                app.UseHsts();
```

Comment:
For simplicity regarding HTTPS configuration, remove the HTTPS and HSTS method calls.

Reply:
done